app/presenters/url.rb

```diff
@@ -0,0 +1,22 @@
+class Url
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers::UrlHelper
+
+  attr_reader :link_text, :path_name, :params
+
+  def initialize(link_text:, path_name:, params: {})
+    @link_text = link_text
+    @path_name = path_name
+    @params = params
+  end
+
+  def to_s
```

Comment:
Right! However I expect `Url#to_s` to act like `URI#to_s` and just return the URL, but in this case.....it does not, it returns a link. Maybe if we renamed the class to `Link` it would make more sense as `Link#to_s` 😁